dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -36,6 +36,16 @@
   {%- endcall %}
 {%- endmacro %}
 
+{% macro set_table_relation_schema(relation, schema) %}
+  {%- if temp_schema is not none -%}
+    {%- set relation = relation.incorporate(path={
+      "schema": schema
+      }) -%}
+      {%- do create_schema(relation) -%}
```

Comment:
```suggestion
  {%- if temp_schema is not none -%}
    {%- set relation = relation.incorporate(path={"schema": schema}) -%}
    {%- do create_schema(relation) -%}
```
let's keep intendations consistent